v2/e2e/configs/egress/v4-nft/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+resources:
+- ../../../../config/default/egress/v4
+
+configMapGenerator:
+- name: coil-config
+  namespace: system
+  files:
+  - cni_netconf=../../../netconf/netconf-kindnet-v4.json
+
+# Adds namespace to all resources.
+namespace: kube-system
+
+# Labels to add to all resources and selectors.
+commonLabels:
+  app.kubernetes.io/name: coil
+
+# [CERTS] Following patches should be uncommented if automatic cert generation is used.
+# patches:
+# - path: ../../../../config/pod/generate_certs.yaml
+#   target:
+#     group: apps
+#     version: v1
+#     kind: Deployment
+#     name: coil-egress-controller
+
+patches:
+  - path: ../coil-egress-controller-nft-patch.yaml
```

Comment:
We need a patch file for coild as well to add `--backend=nftables`.

https://github.com/cybozu-go/coil/pull/346/files#diff-232e8065aa86ddafdf76a92e7afe30fc47788049ea8ea0ac8f0a2b8b93260594R46